doc/hardware/porting/board_porting.rst

```diff
@@ -123,52 +164,71 @@ You need to give your board a unique name. Run ``west boards`` for a list of
 names that are already taken, and pick something new. Let's say your board is
 called ``plank`` (please don't actually use that name).
 
-Start by creating the board directory ``zephyr/boards/<ARCH>/plank``, where
-``<ARCH>`` is your SoC's architecture subdirectory. (You don't have to put your
+Start by creating the board directory ``zephyr/boards/<VENDOR>/plank``, where
+``<VENDOR>`` is your vendor subdirectory. (You don't have to put your
 board directory in the zephyr repository, but it's the easiest way to get
 started. See :ref:`custom_board_definition` for documentation on moving your
 board directory to a separate repository once it's working.)
 
+.. note::
+  A ``<VENDOR>`` subdirectory is mandatory if contributing your board
+  to Zephyr, but if your board is placed in a local repo, then any folder
+  structure under ``<your-repo>/boards`` is permitted.
+  If the vendor is defined in the list in
+  :zephyr_file:`dts/bindings/vendor-prefixes.txt then you must use
+  that vendor prefix as ``<VENDOR>``.
```

Comment:
Should mention the `others` folder as well if a vendor is not defined

Reply:
done